mmseg/core/evaluation/metrics.py

```diff
@@ -70,7 +88,13 @@ def total_intersect_and_union(results, gt_seg_maps, num_classes, ignore_index):
         total_area_pred_label, total_area_label
 
 
-def mean_iou(results, gt_seg_maps, num_classes, ignore_index, nan_to_num=None):
+def mean_iou(results,
+             gt_seg_maps,
+             label_map,
```

Comment:
What's `label_map` for?

Reply:
Look at [here](https://github.com/open-mmlab/mmsegmentation/pull/330/files#diff-82058a6c5d5d710fcc853cd2fbabadf2c6133f91cf5731df05fb03c585f719d5R30), we need ```label_map``` to transform the image that is read from file.